### PR TITLE
Edited for grammar.

### PR DESCRIPTION
**Pull request by proxim66**
Looks good; minor changes.

### CODE REVIEW

File: android/MobiledgeXSDKDemo/README.md [hunk 1]
```diff
@@ -16,80 +16,81 @@
...
 
 #### Cloudlet Details
-In addition to showing details about the cloudlet, this page also lets you initiate a latency test, or a download speed test. The settings for this page let you control things like the number of packets for the latency test, or the size of the download. Click the "Gear" icon to access these settings.
+In addition to showing the details of the cloudlet, this page allows the initiation of a latency test or a download speed test. The settings of this page, allow control of conditions, such as the number of packets for the latency test, or the size of the download. Click the "Gear" icon to access these settings.
 
-You can also choose this cloudlet to be used as the **Edge** server for Face Detection and Face Recognition. Tap the 3-dot menu and select "Use as Face Recognition Edge Host". The **Cloud** server can be set in the same way.
+YYou can also choose this cloudlet to be used as the **Edge** server for Face Detection and Face Recognition. Tap the 3-dot menu and select "Use as Face Recognition Edge Host". The **Cloud** server can be configured in the same way.
```
Comment (brucearmstrong) [inline android/MobiledgeXSDKDemo/README.md]: Typo: Yyou

Comment (proxim66) [reply android/MobiledgeXSDKDemo/README.md]: Thanks for catching that!

File: android/MobiledgeXSDKDemo/README.md [hunk 2]
```diff
@@ -16,80 +16,81 @@
...
 
 ### SDK Calls
 #### Register Client
-This will call [RegisterClient](http://swagger.mobiledgex.net/client/#operation/RegisterClient "RegisterClient") with information that identifies this app's backend software. The session cookie is shown to verify that the call was successful.
+SDK Calls summon the [RegisterClientAPI](http://swagger.mobiledgex.net/client/#operation/RegisterClient "RegisterClient") with information that identifies this app's backend software. The session cookie is shown to verify that the call was successful.
```
Comment (brucearmstrong) [inline android/MobiledgeXSDKDemo/README.md]: "SDK Calls summon the" is pretty awkward. I would suggest making it like the others below. "Register Client will call".

File: android/MobiledgeXSDKDemo/README.md [hunk 3]
```diff
@@ -16,80 +16,81 @@
...
 
 You can move the mobile phone icon by long-pressing on it and dragging it to a new location, or long-pressing anywhere on the map. When you release the icon, you will be presented with two options:
 - Spoof GPS at this location
 - Update location in GPS database.
 
-If you select "Spoof", this new location will be used for subsequent **Find Closest Cloudlet** and **Verify Location** calls. Depending on how close this location is to the location stored in the simulator, your next **Verify Location** call may return a different "accuracy" value, or return a failure. Tapping the mobile phone icon will the distance from the actual location, and any result code available.
+If you select "Spoof," this new location will be used for subsequent **Find Closest Cloudlet** and **Verify Location** calls. Depending on how close the location stored in the simulator is, your next **Verify Location** call may return a different "accuracy" value, or return a failure. Tapping the mobile phone icon will the distance from the actual location, and any result code available.
```
Comment (brucearmstrong) [inline android/MobiledgeXSDKDemo/README.md]: I this really the grammar rule? (School was a long time ago for me :).) I'm using the quotes to represent exactly what is shown on the screen, and there is no comma shown.

File: android/MobiledgeXSDKDemo/README.md (hunk 2 above)
Comment (proxim66) [reply android/MobiledgeXSDKDemo/README.md]: I changed the word from "call" to "summon" to break up the repetitive use that word and to encourage user engagement.

File: android/MobiledgeXSDKDemo/README.md (hunk 3 above)
Comment (brucearmstrong) [inline android/MobiledgeXSDKDemo/README.md]: Should have said "will display the distance".

File: android/MobiledgeXSDKDemo/README.md [hunk 4]
```diff
@@ -16,80 +16,81 @@
...
 
-If you select "Update location", this new location will be sent to the location simulator and will be used until another location is sent, or "Reset Location" is performed. This allows you to simulate having your real location be anywhere in the world, and you can verify that the expected cloudlet is found by **Find Closest Cloudlet**.
+If you select "Update location," this new location will be sent to the location simulator and will be used until another location is sent, or "Reset Location" is performed. Reset Location allows you to simulate your real location be anywhere in the world, and you can verify that **Find Closest Cloudlet** finds the expected cloudlet.
```
Comment (brucearmstrong) [inline android/MobiledgeXSDKDemo/README.md]: "simulate your real location be anywhere" doesn't quite make sense. Maybe "being anywhere"?

File: android/MobiledgeXSDKDemo/README.md (hunk 2 above)
Comment (brucearmstrong) [reply android/MobiledgeXSDKDemo/README.md]: But it is literally a "function call" or "SDK call". The plural "SDK Calls" would be fine for an overview of the different calls, but this line refers to a singular "call".
"XYZ will call" is repeated 4 times below, which I think is fine for this kind of doc.

File: android/MobiledgeXSDKDemo/README.md [hunk 5]
```diff
@@ -16,80 +16,81 @@
...
 
 Things to try:
 - You may switch between the front and rear camera by tapping the camera icon.
-- Tap the 3-dot menu, and select "Play Video" to process a canned video instead of images from the camera. This is useful for unattended demos or benchmarking.
+- Tap the 3-dot menu, and select "Play Video" to process a canned video instead of images from the camera. This function is useful for unattended demos or benchmarking.
 - Go to settings and turn on "Show Latency Stats after session" to get a stats summary that can be copy/pasted for additional use.
 
 #### Face Detection
-The Face Detection activity provides a visual comparison of the latency provided by an Edge cloudlet vs. that of a server in the public cloud. The Edge cloudlet can be determined in a few ways, in this order of priority: 
+TThe Face Detection activity provides a visual comparison of the latency offered by an Edge cloudlet vs. that of a server in the public cloud. The Edge cloudlet can be determined in a few ways, in this order of priorit:
```
Comment (brucearmstrong) [inline android/MobiledgeXSDKDemo/README.md]: Another typo: "TThe"

File: android/MobiledgeXSDKDemo/README.md (hunk 3 above)
Comment (proxim66) [reply android/MobiledgeXSDKDemo/README.md]: Quotation Marks:  We use quotation marks with direct quotes, with titles of certain works, to imply alternate meanings, and to write words as words.
Block quotations are not set off with quotation marks.
The quoted text is capitalized if you’re quoting a complete sentence and not capitalized if you’re quoting a fragment.
_Do commas and periods go inside or outside quotation marks?_
Commas and periods always go inside the quotation marks in American English; dashes, colons, and semicolons almost always go outside the quotation marks; question marks and exclamation marks sometimes go inside, sometimes stay outside.

Comment (proxim66) [reply android/MobiledgeXSDKDemo/README.md]: Technically, if you use a quote of any kind the writer is  encouraged to cite their source to allow the reader to determine if the quote has been captured correctly.  But, in your favor, I do have a bunch of things happening and I missed a few corrections. Sorry :)

Comment (lgarner) [reply android/MobiledgeXSDKDemo/README.md]: *cough* I'm mildly conflicted. It's correct English grammar, but will fail to compile in a coder's mind given it's a guide for new developers. Anyway, carry on everyone :)

File: android/MobiledgeXSDKDemo/README.md (hunk 1 above)
Comment (proxim66) [reply android/MobiledgeXSDKDemo/README.md]: Will fix.

**Comment by brucearmstrong** [comment]
@proxim66, do you plan to merge this?

**Pull request merged**